ClientApp/src/Styles/constants.scss

```diff
@@ -0,0 +1,81 @@
+﻿@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```

Comment:
I don't think we want all the css in here especially pieces that we aren't using yet, I'd rather keep it quite basic and then add our own styling to this project when we agree on it - so just hand pick the parts contributing to your form styling for now